components/connector/internal/gqlschema/schema.graphql

```diff
@@ -0,0 +1,59 @@
+# Token
+type Token {
+    url: String! # eg.: "https://connector-service.test.cluster.kyma.cx/v1/applications/info?token=1edfc34g"
+    token: String! # eg.: "1edfc34g"
+}
+
+# Certificate
+type Certificate {
+    crt: String!
+    caCrt: String!
+    clientCrt: String!
+}
+
+# MPInfo
+type MPInfo {
```

Comment:
hmm, what about `ManagementPlaneConfiguration`, `ManagementPlaneDetails`, or something like that?

Reply:
I changed it to ManagementPlaneInfo